dev/breeze/doc/09_release_management_tasks.rst

```diff
@@ -189,6 +189,26 @@ These are all of the available flags for the ``release-prod-images`` command:
   :width: 100%
   :alt: Breeze release management release prod images
 
+Adding git tags for providers
+"""""""""""""""""""""""""""""
+
+This command can be utilized to manage git tags for providers within the airflow remote repository during provider releases.
+Sometimes in cases when there is a connectivity issue to Github, it might be possible that local tags get created and lead to annoying errors.
+The default behaviour would be to clean such local tags up.
+
+If you want to disable this behaviour, set the env CLEAN_LOCAL_TAGS to false.
```

Comment:
You also need to add about the new click option here: flag and env variable

Reply:
yep

Reply:
Added this change